ingest/src/server_5g.rs

```diff
@@ -183,16 +175,19 @@ pub async fn grpc_server(shutdown: triggered::Listener, server_mode: String) ->
         speedtest_report_tx,
     )?;
 
-    let api_token = std::env::var("API_TOKEN").map(|token| {
-        format!("Bearer {}", token)
-            .parse::<MetadataValue<_>>()
-            .unwrap()
-    })?;
+    let api_token = settings
```

Comment:
I thought we’d removed all this api bearer token auth stuff for the ingest server?

Reply:
I guess this is just for the 5g server which _is_ still authenticated?

Reply:
Correct